types/chrome-apps/test/index.ts

```diff
@@ -324,5 +324,40 @@ function testSystemNetwork() {
     });
 }
 
-import webview = chrome.webview;
-let element: webview.HTMLWebViewElement;
+const gcmMessage = <chrome.gcm.OutgoingMessage>{};
+gcmMessage.data = {
+    /*goog: 'any', should not be allowed, and it is not :) */
+    test: true
+};
+
+let wve: chrome.webview.HTMLWebViewElement = (<any>document.getElementById('webview'));
+wve.name = 'test';
+wve.src = 'http://github.com/niikoo'
```

Comment:
Might be better to add the repo site here instead incase your username changes etc
http://github.com/DefinitelyTyped

Reply:
Thank you for your feedback.
It's now changed and committed.